src/Public/Layouts.ps1

```diff
@@ -33,7 +33,8 @@ function New-PodeWebCell
     param(
         [Parameter(Mandatory=$true)]
         [hashtable[]]
-        $Content
+        $Content,
+        [ValidateRange(1, 12)][int]$Size
```

Comment:
Split this one onto separate lines, with a Parameter attribute (so it's consistent with the rest):

```powershell
[Parameter()]
[ValidateRange(1, 12)]
[int]
$Size
```

I'm wondering if calling the parameter `$Width` might be more fitting, for if we ever need to add a `$Height` at some point 🤔

Reply:
I'll fix this tomorrow and renaming it to $Width sounds like a good idea so I'll do that as well.